### ISSUE TITLE
Number literals in tag field cause tasks to fail

### ISSUE THREAD

**Issue by seadowg** [concourse/docker-image-resource#43]
Using an image resource defined in my task like this:

``` yml
image_resource:
  type: docker-image
  source:
    repository: java
    tag: 8
```

I get the following error when my task runs:

```
resource script '/opt/resource/check []' failed: exit status 1

stderr:
failed to read request: json: cannot unmarshal number into Go value of type string
```

Works fine if I surround the `8` with quotes so it infers that the value is a string rather than a number. As far as I'm aware however a docker tag will always be a string so it would be nice if the resource didn't trip over this.

**Comment by DenisIzmaylov** [comment]
Up! I have the same for:
```
image_resource:
  type: docker-image
  source: {repository: node, tag: 5}
```

**Comment by chendrix** [comment]
Fixed by concourse/docker-image-resource#89
